blueprints/app/files/bower.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
I'm tempted to move this into a different generator rather than just omitting it. `bower init` is _way_ overkill for something that will never be run as a `bower` package.

```
$ bower install --save 'jquery'
bower cached        https://github.com/jquery/jquery-dist.git#3.1.1
bower validate      3.1.1 against https://github.com/jquery/jquery-dist.git#*
bower no-json       No bower.json file to save to, use bower init to create one
```

(For people manually installing `bower` packages instead of transitively via `ember install`.)